era/release-config.yml

```diff
@@ -30,8 +30,8 @@ repositories:
         - regex_replacer: 
             { 
               file_path_relative: "pyproject.toml",
-              regex: 'version = "0.4.0.dev0"',
-              replacement: 'version = "0.4.0.dev0"',
+              regex: 'version = "0.1.0"',
```

Comment:
Think this is not used as intended.

Reply:
I would keep it this way and for the next release it will be moved to 0.2.0. Or what would be your suggestion?